lib/shared/addon/components/search-text/component.js

```diff
@@ -0,0 +1,46 @@
+import Component from '@ember/component';
+import { set } from '@ember/object';
+import layout from './template';
+import { matches } from 'shared/components/sortable-table/component';
+
+const SEARCH_FIELDS = ['displayName', 'id:prefix', 'displayState'];
+
+export function filter(array, searchText, searchFields = SEARCH_FIELDS) {
```

Comment:
This looks to be copied from sortable-table (and apparently `pod-dots` and `namespace-table` do the same thing)?  I'd like to see is pulled out as its own utility and then all components use that instead of adding a 4th copy-paste of the logic to filter an array by search expression.